src/rdkafka_conf.c

```diff
@@ -437,6 +437,21 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
      1, 1000000, 1000000},
     {_RK_GLOBAL, "max.in.flight", _RK_C_ALIAS,
      .sdef = "max.in.flight.requests.per.connection"},
+    {_RK_GLOBAL, "metadata.recovery.strategy", _RK_C_S2I,
+     _RK(metadata_recovery_strategy),
+     "Controls how the client recovers when none of the brokers known to it "
+     "is available. If set to `none`, the client fails with a fatal error. "
```

Comment:
We should explain about default case i.e. `rebootstrap` first.

Reply:
I've taken this directly from Java client documentation, as it's well explained and applies to librdkafka as well.